x-pack/plugin/deprecation/src/main/java/org/elasticsearch/xpack/deprecation/NodeDeprecationChecks.java

```diff
@@ -169,6 +168,17 @@ static DeprecationIssue watcherNotificationsSecureSettingsCheck(Settings nodeSet
         return null;
     }
 
+    static DeprecationIssue watcherHipchatNotificationSettingsCheck(Settings nodeSettings, PluginsAndModules plugins) {
+        if (nodeSettings.getByPrefix("xpack.notification.hipchat.").size() > 0) {
+            return new DeprecationIssue(DeprecationIssue.Level.CRITICAL,
+                "Watcher Hipchat notifications will be removed in the next major release",
+                "https://www.elastic.co/guide/en/elasticsearch/reference/master/breaking-changes-7.0.html" +
```

Comment:
```suggestion
                "https://www.elastic.co/guide/en/elasticsearch/reference/7.0/breaking-changes-7.0.html" +
```

This link is currently broken due to the 7.0 breaking changes not being in the `master` docs anymore, as are most of the others. I'll have a PR up shortly fixing the others, but this one should switch before it's merged if possible.

Reply:
Thanks Gordon